main/main.tf

```diff
@@ -0,0 +1,20 @@
+provider "azurerm" {
+    features {}   
+}
+resource "azurerm_resource_group" "rg" {
+    name = var.rg_name
+    location = var.location
+    tags = var.tags
+}
+
+resource "azurerm_storage_account" "rg" {
+  name                     = "sandystorage4877"
+  resource_group_name      = azurerm_resource_group.rg.name
+  location                 = azurerm_resource_group.rg.location
+  account_tier             = "Standard"
```

Comment:
why is this hard coded, move this to variables

Reply:
fixed

Reply:
you have not fixed... you need to move this into varialbes.tf... this applies to all !